### PR TITLE
feat: create pages group1 login and signup page

### LINKED ISSUE #183: Create pages: `Group1` Login and signup Page
## Description

- We need to create `Login` and `Sign up` pages. In this issue, do not need to consider about logics.

## Criteria

image

- [ ] `/`, `/login`
- [ ] `/signup`
- [ ] `forgot-password`
- [ ] `reset-password`

### PR DESCRIPTION
**Pull request by Akikaze1119**
## Overview
I created login and signup pages

## Changes
- implement styles and form function in `/`, `/login`.
- create `/signup`, `/forgot-password`, `/reset-password` page.
- create `Logo` component.
- create a password validation function.
- fix the tab index of password input. `/components/ui/input.tsx`
- remove `capitalize` class from `/components/ui/button.tsx`

## Review points
- [x] Ensure the UI is properly implemented.
- [x] Form input validation is properly implemented.

## Screenshots or videos
https://github.com/Tascurator/tascurator-frontend/assets/132441773/f33cf92b-2f2c-4102-acd8-5524ac4fb16e

## Notes
`/` page will be implemented in `📺Implement logic` phase.
 *redirect depending on login status. (After login: `/sharehouses/`, Before login: `/login` )

[milestone](https://www.notion.so/cdc9aa0a78b842659d49b804353f031f?v=58088810df0e42d2b44520f806bb90b8&p=de081b50435e4434bf077c18662faa3f&pm=s)

## Assignee Checklist:
- [x] The base branch is correct (no accidental merges)
- [x] The branch name follows our branch naming rules
- [x] The PR title follows our PR title rules
- [x] My code follows our coding style
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [x] Code readability and simplicity
- [x] Follows best practices and coding standards
- [x] Understandable and maintainable code

### CODE REVIEW

**Comment by yuki-92c** [comment]
Thank you Satomi for your hard work! I have some feedbacks so please check them and consider about changes :)

**Comment by yuki-92c** [comment]
- Logo disappears on the "Reset Password" and "Fogot Password" screens for smart phone size

IMG_9843
IMG_9844

**Comment by yuki-92c** [comment]
- Error message seems like doesn’t match to the actual error

test@test com

**Comment by yuki-92c** [comment]
- the text below the input box looks not suitable 
  - Less than or equal to 8 characters long. => **Greater** than or equal to 8 characters long.
  - Greater than or equal to 30 characters long.  => **Less** than or equal to 30 characters long.
  - Also, I feel like it is a bit odd to see `checked` even I don’t input any letters.

Password

File: src/components/reset-password/Form.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+          />
+          {errors.password?.message && (
+            <FormMessage message={errors.password.message} />
+          )}
+        </div>
+        <div className={'flex flex-col mb-3'}>
+          <Input
+            id="confirmPassword"
+            type="password"
+            label="ConfirmPassword"
```
Comment (yuki-92c) [inline src/components/reset-password/Form.tsx]: `ConfirmPassword`  => `Confirm password`

**Comment by Akikaze1119** [comment]
@yuki-92c 
Sorry, I forgot to leave comments about this.
After discussing with Rina, we've decided to hide the logo on mobile views. (It will still appear on tablet and laptop sizes.)

**Reason:**
We believe the logo is not essential on this page. 
Additionally, it causes the page to elongate, requiring users to scroll to see input areas and buttons

> * Logo disappears on the "Reset Password" and "Fogot Password" screens for smart phone size

**Comment by Akikaze1119** [comment]
@yuki-92c 
Thank you for pointing that out. I've corrected the error in the Zod schema for signup and reset-password

```    .min(PASSWORD_MIN_NUMBERS, minLength('Password', PASSWORD_MIN_NUMBERS))```
↓
```    .min(PASSWORD_MIN_LENGTH, minLength('Password', PASSWORD_MIN_LENGTH))```


> * Error message seems like doesn’t match to the actual error

**Comment by Akikaze1119** [comment]
> * the text below the input box looks not suitable
>   
>   * Less than or equal to 8 characters long. => **Greater** than or equal to 8 characters long.
>   * Greater than or equal to 30 characters long.  => **Less** than or equal to 30 characters long.
>   * Also, I feel like it is a bit odd to see `checked` even I don’t input any letters.

I fixed this issue! I've merged both into a single constraint statement.
`Between 8 and 30 characters long.`

**Review by yuki-92c — APPROVED** [review]
Thank you so much for your hard work, Satomi! LGTM

**Pull request merged**